rr/ai/ai_manager.py

```diff
@@ -87,6 +87,9 @@ def __init__(
         self.postprocess_obj = PostProcessDetection(
             model, disp_width, disp_height)
 
+        # Make a warmup run to start the engine
+        self.inference_obj.run(np.zeros((1, 300, 300, 3), dtype='float32'))
```

Comment:
Isn't the model dependant on width and height? is it ok to use a fixed 300x300 image?